src/styles/stopwatch/stopwatch.scss

```diff
@@ -0,0 +1,41 @@
+.main {
+  @include horizontal-center-flex;
+
+  min-height: 100vh;
+
+  margin: 0;
+}
+
+.stopwatch {
+  @include horizontal-center-flex;
+  @include circle($watch-size);
+
+  position: relative;
```

Comment:
you can't use positioning for BEM blocks

Reply:
@roman-zherebetskyi In this case, he can for a reason that this does not affect the neighbouring blocks, it's just to position children's elements